ui/helpers/url.js

```diff
@@ -20,8 +26,14 @@ export const getApiUrl = function getApiUrl(uri) {
   return getServiceUrl(`/api${uri}`);
 };
 
+// bugs can be one bug or a comma separated (no spaces) string of bugs
+export const bugzillaBugsApi = function bugzillaBugsApi(api, params) {
+  const query = createQueryParams(params);
+  return `${bzBaseUrl}${api}${query}`;
+};
+
 export const getBugUrl = function getBugUrl(bug_id) {
-  return `https://bugzilla.mozilla.org/show_bug.cgi?id=${bug_id}`;
+  return bugzillaBugsApi('show_bug.cgi', { id: bug_id });
```

Comment:
I wonder if we should pick a different name for `bugzillaBugsApi()` if we're calling it for non-API cases too? Or else have `bugzillaBugsApi()` stay the same name, but have it add the `/rest/` part to the URL and make it API-only?

Reply:
Hmm, you're totally right.  The ``show_bug`` isn't an API.  My bad.  I like your idea of including ``rest/`` in the Api function.  How's this look?  :)